splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/PreparedStatementIT.java

```diff
@@ -35,6 +37,7 @@
  * @author Jeff Cunningham
  *         Date: 6/19/13
  */
+@Category(SerialTest.class) // Not run in parallel since it is changing global DB configs
```

Comment:
Instead of marking the entire suite as serial, potentially increasing the build time, does it make sense to move this system-property test another serial test suite (or have a dedicated serial test suite only for those properties).

Reply:
Good idea!

Reply:
Tracked by DB-11618